native/src/main/java/io/ballerina/stdlib/graphql/runtime/engine/Engine.java

```diff
@@ -247,6 +249,20 @@ private static List<String> getPathList(BArray pathArray) {
         return result;
     }
 
+    public static Object getMethod(BObject service, BString methodName) {
+        ServiceType serviceType = (ServiceType) TypeUtils.getType(service);
+        return getMethod(serviceType, methodName.getValue());
+    }
+
+    private static MethodType getMethod(ServiceType serviceType, String methodName) {
+        for (MethodType remoteMethod : serviceType.getMethods()) {
+            if (remoteMethod.getName().equals(methodName)) {
```

Comment:
Can these method names be null?

Reply:
Those cannot be null. We could change the logic other-way around `methodName.equals(serviceMethod.getName())` for safety.